### ISSUE TITLE
Governance

### ISSUE THREAD

**Issue by sebmck** [yarnpkg/yarn#274]
@wycats has some thoughts around this.

**Comment by doug-wade** [comment]
I am curious to know where this stands in light of https://github.com/yarnpkg/rfcs/pull/1#issuecomment-273318428.  It sounds like the situation as it currently stands is "let's hold off on new major features while we figure out a sustainable governance model and stabilize the fb use case" which make a _lot_ of sense to me, and I wholeheartedly support; however, as someone who would like to use yarn more extensively, I'd be curious to know how the roadmap will be handled, how rfcs make it through to features, and the like.  Please don't feel harried or anything, just a gentle ping :smile_cat:.

**Comment by destroyerofbuilds** [comment]
In yarnpkg/rfcs#1, @cpojer mentioned the desire of the current `yarn` core team at Facebook to build a `solid community driven core team`. As a community member interested in the shared ownership of `yarn`, and involvement of the wider JavaScript community in its development and support, does Yarn publish a core team page similar to Ember - http://emberjs.com/team/

**Comment by Daniel15** [comment]
> does Yarn publish a core team page

There's the core team page on GitHub (https://github.com/orgs/yarnpkg/teams/core), looks like the team is parked as private for some reason though. @cpojer @kittens is there a reason the yarnpkg/core team is private?

**Comment by cpojer** [comment]
I made some adjustments to the teams to reflect the current state and turned the core group visible.

**Comment by saper** [comment]
I am not sure it works very well for https://github.com/libuv/leps 

it kind of may discourage from hacking and exploring risky opportunities

also, should RFCs include working code (basics for the IETF work)?

**Comment by KidkArolis** [comment]
Given there have been a bunch of RFCs implemented and accepted - should this issue be closed now?

**Comment by wycats** [comment]
@KidkArolis indeed! I'm psyched that this ended up being the Yarn governance ❤️

**Comment by olingern** [comment]
Not to dig an old issue up, but ...

I asked over in the v2 release who the governing body may or may not be ( _it's currently unclear, to me at least_ ), but seeing as how yarn v2 is going to potentially replace yarn v1, I'm interested in how the governance works here?

**Comment by wwahammy** [comment]
Hey all, I don't know if this is addressed but it looks like PR's for known issues aren't being addressed on Yarn v1. Without knowing the governance model and who is in charge of what, it kind of looks like no one.

Is there an individual or group in charge of handling issues and PRs on Yarn?

**Comment by olingern** [comment]
@wwahammy I volunteered my time on a different issue, but it was kind of unclear what that looked like in practice. There was a bit of a rift between yarn 1 and yarn 2's release, so I thought it was best to let that settle.

I general, I think Yarn v1 should be maintenance only and no new features added. That's what Yarn v2 is for. So, I've pushed back pretty hard on PR's, like [this one](https://github.com/yarnpkg/yarn/pull/7663#issuecomment-644472197) where a number of Microsoft employees wanted something that was easily accomplished outside of Yarn.

The number of duplicates, low quality issues, and un-mergable PR's is unmanageable for one, probably even two people. No idea on how to deal with that, other to auto-close everything new.

Anyway, I'd be curious to see if @cpojer or @arcanis had any thoughts here. Governance / managing the project.
